apps/common/lib/lexical/ast.ex

```diff
@@ -87,6 +87,18 @@ defmodule Lexical.Ast do
   @type short_alias :: atom()
   @type alias_segments :: [short_alias]
 
+  @doc """
+  Guard used to determined whether an AST node is a call.
+
+  ## Example
+
+      def my_fun({call, _meta, args}) when is_call(call, args), do: ...
+
+  """
+  @non_call_nodes [:., :__aliases__, :__block__, :"::", :{}, :|>, :%, :%{}]
```

Comment:
I'm curious where you got this list from?

Reply:
I got the idea from ElixirSense, though ElixirSense doesn't contain `:__block__` which I think is incorrect. That said, I'd actually like for this (and others) to live in `Sourceror` (see [this issue](https://github.com/doorgan/sourceror/issues/105)), so it's possible that this will be removed prior to merging.